lib/src/disk_usage/disk_usage_view.dart

```diff
@@ -48,23 +51,61 @@ class _DiskUsageViewState extends State<DiskUsageView> {
         label: const Text("Export Logs"),
         icon: const Icon(Icons.bug_report),
         onPressed: () async {
-          // i would use "saveFile" but for some reason it crashes
-          final dir = await FilePicker.platform.getDirectoryPath();
-          if (dir == null) return;
-          final path = p.join(dir, "ditto_log.txt");
-          await DittoLogger.exportLogs(path);
-          _showSnackbar("Logs exported to $path");
+          try {
+            String exportPath;
+            
+            if (Platform.isIOS) {
+              // On iOS, export to the app's Documents directory
+              final documentsDir = await getApplicationDocumentsDirectory();
+              exportPath = p.join(documentsDir.path, "ditto_log.txt");
+            } else {
+              // On Android, macOS, and other platforms, let user choose directory
+              final dir = await FilePicker.platform.getDirectoryPath();
+              if (dir == null) return;
+              exportPath = p.join(dir, "ditto_log.txt");
+            }
+            
+            await DittoLogger.exportLogs(exportPath);
+            
+            if (Platform.isIOS) {
+              _showSnackbar("Logs exported to Files app\n(Check in Files > On My iPhone > Example)");
+            } else {
+              _showSnackbar("Logs exported to $exportPath");
+            }
+          } catch (e) {
+            _showSnackbar("Export failed: ${e.toString()}");
+          }
         },
       );
 
   Widget get _exportData => OutlinedButton.icon(
         label: const Text("Export Data Directory"),
         icon: const Icon(Icons.folder),
         onPressed: () async {
-          final path = await FilePicker.platform.getDirectoryPath();
-          if (path == null) return;
-          copyDir(widget.ditto.persistenceDirectory, path);
-          _showSnackbar("Data exported to $path");
+          try {
+            String? exportPath;
+            
+            if (Platform.isIOS) {
```

Comment:
We can't use `saveFile` here because we need a directory, and web APIs don't support that, but many of the other commends about not using specific native-only imports apply here too